tekton/utils/pipeline.go

```diff
@@ -46,7 +46,7 @@ type Pipeline struct {
 	// ServiceAccount is the ServiceAccount to use during the execution of the Pipeline
```

Comment:
Can you change the first word here to say ServiceAccountName?

Reply:
Fixed